alkira/resource_alkira_service_infoblox.go

```diff
@@ -230,6 +230,11 @@ func resourceAlkiraInfoblox() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
+			"allow_list_id": {
```

Comment:
What's the naming of the field on UI for this? It's called `allow list`? It's confusing, since we never have a resource called `allow list` anywhere.

Reply:
hmmmmm, let me confirm with UI folk... since if we call it whitelist_prefix_list, that's also not there...